Dockerfile

```diff
@@ -1,6 +1,6 @@
 ## docker image is intended for development environments only.
 ## to use snowshu in production install the package via pip
-FROM python:3.7-slim-buster
+FROM python:3.8-slim-buster
```

Comment:
This is not needed as I removed usage of walrus operators - should I revert this change?

Reply:
Let's revert it for now since the `setup.py` seems to only expect python version 3.7. We can work on upgrading/updating that dependency later.

Reply:
done